src/open_inwoner/openzaak/utils.py

```diff
@@ -140,3 +141,14 @@ def get_retrieve_resource_by_uuid_url(
         client.schema, operation_id, base_url=client.base_url, **path_kwargs
     )
     return url
+
+
+def get_statustype_url_from_status_url(status_url: str) -> str:
+    """
+    Get statustype url by appending the "rpath" of `status_url` (the part
+    after the last '/') to the root url of the catalogi API
+    """
+    CATALOGI_ROOT = "https://catalogi.nl/api/v1/"
```

Comment:
I'm not sure why this is hardcoded, also if I understand it correctly I don't think this function would give me the correct statustype url, because the UUID for a `Status` is not the same as the UUID for its `StatusType`.

I think you can leave this function out and do something like this in `src/open_inwoner/openzaak/notifications.py`:

```python
from open_inwoner.openzaak.cases import resolve_status

...

# check status notification setting on `ZaakTypeStatusTypeConfig`
resolve_status(case)
statustype_url = case.status.statustype
```